execute/tokendata/usdc/http_test.go

```diff
@@ -64,7 +65,7 @@ func Test_NewHTTPClient_New(t *testing.T) {
 func Test_HTTPClient_Get(t *testing.T) {
 	tt := []struct {
 		name               string
-		getTs              func() *httptest.Server
+		getTs              func(t *testing.T) *httptest.Server
```

Comment:
Why this change? I don't see the `testing.T` used anywhere in these funcs.

Reply:
The `default timeout` case uses a `require.NoError`

Reply:
Oh I see, they use the top level `t *testing.T` in the test func. I usually rename nested `testing.T` to `tt` but thats just preference.

Reply:
Hmmm I think using a different name could be error prone, because the original `t` is still in scope to be referenced accidentally. This happens with ctx and err often as well.